### DISCUSSION TITLE
Configuring vector connection to kafka

### DISCUSSION THREAD

**Discussion by k8s-comandante** [vectordotdev/vector#10102] [Q&A] [answered]
**Hello everyone!**

I have a Kafka cluster in my company that runs on the kerberos keytab, I want to use a vector agent in order to pick up microservices logs from Kafka,   in the documentation https://vector.dev/docs/reference/configuration/sources/kafka/#:~:text=message_key-,librdkafka_options,-optional this case is hardly touched upon and hence there is a misunderstanding of how to implement it. Tried to configure sources like this but nothing happens:

**[sources.kafka-logs.librdkafka_options]
"sasl.kerberos.keytab" = "/opt/file.keytab"**

also tried https://component-pages--vector-project.netlify.app/docs/reference/sources/kafka/#:~:text=%23-,Advanced,-librdkafka_options.%22client.id

**librdkafka_options. "sasl.kerberos.keytab" = "/opt/file.keytab"**

but also unsuccessfully.

**Comment by jszwedko** [comment, marked as answer]
Hi @k8s-comandante ! Both of those should be correct ways to configure the rdkafka options. What makes you think they are not working?

**Reply by k8s-comandante** [reply]
Hey @jszwedko  !
I specify a non-existent keytab file and the vector does not even say about it.

**Reply by jszwedko** [reply]
Ah I see. Looking at the [configuration options page for librdkafka](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) it seems like that value is only used in templating `sasl.kerberos.kinit.cmd`. Is it possible that there needs to be additional librdkafka configuration to use keberos? I see some additional documentation here: https://github.com/edenhill/librdkafka/wiki/Using-SASL-with-librdkafka that could be useful.
